packages/components/upload/src/use-handlers.ts

```diff
@@ -100,7 +100,7 @@ export const useHandlers = (
         props.onError(err as Error, uploadFile, uploadFiles.value)
       }
     }
-    uploadFiles.value = [...uploadFiles.value, uploadFile]
+    Object.assign(uploadFiles.value, [uploadFile])
```

Comment:
I think it's not the correct solution. Any difference with `uploadFiles.value[0] = uploadFile`?

It will overwrite the first element instead of adding it.

Reply:
I have fixed it

Reply:
I am fixing now